.gitignore

```diff
@@ -13,3 +13,4 @@ dist/
 ./.packages
 .coverage
 gen_docs/
+.vscode/settings.json
```

Comment:
Happy to take this out and commit the file, my settings:
```json
{
    "editor.formatOnSave": true,
    "[python]": {
        "editor.defaultFormatter": "ms-python.black-formatter"
    }
}
```

Reply:
I would suggest to separate it from main changes. I'm fine with adding your settings to repo, it should not hurt.

Reply:
Will do!